### ISSUE TITLE
[Accessibility]: Faster navigation to ref panel

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36623]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

It is time consuming to navigate to the ref panel after opening a URL pointing to a page with an open ref panel.

(This point also applies to keyboard navigation.)

### Expected behavior

You can quickly navigate to the ref panel. We should perhaps use skip links or similar for this.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by varungandhi-src** [comment]
Somewhat related, but in the "opposite" direction:
- https://github.com/sourcegraph/sourcegraph/issues/36581

**Comment by umpox** [comment]
Might be fixed by https://github.com/sourcegraph/sourcegraph/issues/36581#issuecomment-1175015321
